Karma-Khameleons/bangazon-website
# Rw shoppingcart

## Description
This pull request covers the addition of a users shopping cart total across the relevant views.

## Related Ticket(s)
#36 
#7


## Problem to Solve
Users needed to be able to see there shopping cart total in the navbar

## Changes
added try and except to each necessary view in order test wether a customer had an order and if so display the current total in the navbar.


## Expected Behavior
Product total should be visible in each user authenticated view


## Steps to Test Solution
```
git fetch
git checkout rw-shoppingcart
```

login or register a user.
create an order
see that order total updates


## Unit Testing
- [] I certify that all existing tests pass

## Documentation
